libs/data-mapper/src/lib/models/Function.ts

```diff
@@ -41,6 +41,7 @@ export enum FunctionCategory {
   Math = 'Math',
   String = 'String',
   Utility = 'Utilities',
+  Custom = 'Custom',
```

Comment:
Add a comment that this is deliberately last since it's position contradicts the above comment about them needing to be in alphabetic order